pages/linux/schroot.md

```diff
@@ -0,0 +1,25 @@
+# schroot
+
+> Run command or interactive shell with special root directory. More customizable than standard chroot.
+> Source code: https://gitlab.com/codelibre/schroot.
+> More information: <https://wiki.debian.org/Schroot>.
+
+- Run command in a specific chroot:
+
+`schroot --chroot {{chroot}} {{command}}`
+
+- Pass options to a command in a specific chroot:
+
+`schroot --chroot {{chroot}} {{command}} -- {{options for command}}`
+
+- Run command in all available chroots:
```

Comment:
When a chroot could be unavailable?
```suggestion
- Run a command in available chroots:
```

Reply:
True, chroots either do or don't exist, I think it's important to tell the user it's in _all_ chroots though
```suggestion
- Run a command in all chroots:
```

Reply:
Ok, then:
```suggestion
- Run a command in all available chroots:
```